docs/commands/json.mset.md

```diff
@@ -1 +1,60 @@
-Sets or updates the JSON value of one or more keys.
+Set or updates JSON values of one or more keys (multiple) for a given tripplet `key-path-value`
```

Comment:
Set or update one or more JSON values according to the specified `key`-`path`-`value` triplets.